hoover/contrib/oauth2/Readme.md

```diff
@@ -0,0 +1,5 @@
+### setup
+* add `hoover.contrib.oauth2` to `INSTALLED_APPS`
+* set `HOOVER_OAUTH_LIQUID_URL` to the URL of the liquid users app
+* set `HOOVER_OAUTH_LIQUID_CLIENT_ID` and `HOOVER_OAUTH_LIQUID_CLIENT_SECRET`
```

Comment:
Where can one get the `HOOVER_OAUTH_LIQUID_CLIENT_ID` and `HOOVER_OAUTH_LIQUID_CLIENT_SECRET` keys?
It'd be really nice to have the documentation here a bit more complete and have link to the [liquid-core documentation](https://github.com/hoover/liquid-core/tree/master/liquidcore/site)? (when you'll write it, that is)

At least a reference to the `liquid-code / Applications / Authentication` admin would be nice.